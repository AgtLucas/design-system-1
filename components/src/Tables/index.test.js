import * as TableComponents from './';

describe('Table', () => {
  it('works', () => {
    expect(TableComponents).toBeDefined();
    expect(TableComponents).toMatchSnapshot();
  });
});