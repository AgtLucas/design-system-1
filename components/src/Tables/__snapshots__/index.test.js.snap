// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Table works 1`] = `
Object {
  "ActionCell": [Function],
  "Body": [Function],
  "Cell": [Function],
  "CreateRow": [Function],
  "CreateRowCell": [Function],
  "Header": [Function],
  "HeaderCell": [Function],
  "Row": [Function],
  "Table": [Function],
}
`;
